client/src/components/forms/MoodForm.js

```diff
@@ -1,7 +1,274 @@
+  const [exercise, setExercise] = useState(null);
+  const [exerciseTime, setExerciseTime] = useState(null);
+  const [yoga, setYoga] = useState(null);
+  const [yogaTime, setYogaTime] = useState(null);
+  const [meditation, setMeditation] = useState(null);
+  const [meditationTime, setMeditationTime] = useState(null);
+
+
+  const excercizeTimeInput = exercise => {
+    if (exercise === true) {
+      return (
+        <FormControl className={classes.formControl}>
+          <InputLabel id="exercise-time-attribute">Time Exercised</InputLabel>
+          <Select
+            labelId="exercise-time-attribute"
+            id="exercise-time-attribute"
+            value={exerciseTime}
+            onChange={handleExerciseTimeChange}
+          >
+            <MenuItem value={10}>10 min</MenuItem>
+            <MenuItem value={20}>20 min</MenuItem>
+            <MenuItem value={30}>30 min</MenuItem>
+            <MenuItem value={40}>40 min</MenuItem>
+            <MenuItem value={50}>50 min</MenuItem>
+            <MenuItem value={60}>60 min</MenuItem>
+          </Select>
+        </FormControl>
+      );
+    } else {
+    }
+  };
+
+  const yogaTimeInput = yoga => {
+    if (yoga === true) {
+      return (
+        <FormControl className={classes.formControl}>
+          <InputLabel id="yoga-time-attribute">Yoga Time</InputLabel>
+          <Select
+            labelId="yoga-time-attribute"
+            id="yoga-time-attribute"
+            value={yogaTime}
+            onChange={handleYogaTimeChange}
+          >
+            <MenuItem value={10}>10 min</MenuItem>
+            <MenuItem value={20}>20 min</MenuItem>
+            <MenuItem value={30}>30 min</MenuItem>
+            <MenuItem value={40}>40 min</MenuItem>
+            <MenuItem value={50}>50 min</MenuItem>
+            <MenuItem value={60}>60 min</MenuItem>
+          </Select>
+        </FormControl>
+      );
+    } else {
+    }
+  };
+
+  const meditationTimeInput = meditation => {
+    if (meditation === true) {
+      return (
+        <FormControl className={classes.formControl}>
+          <InputLabel id="meditation-time-attribute">Meditation Time</InputLabel>
+          <Select
+            labelId="meditation-time-attribute"
+            id="meditation-time-attribute"
+            value={meditationTime}
+            onChange={handleMeditationTimeChange}
+          >
+            <MenuItem value={10}>10 min</MenuItem>
+            <MenuItem value={20}>20 min</MenuItem>
+            <MenuItem value={30}>30 min</MenuItem>
+            <MenuItem value={40}>40 min</MenuItem>
+            <MenuItem value={50}>50 min</MenuItem>
+            <MenuItem value={60}>60 min</MenuItem>
+          </Select>
+        </FormControl>
+      );
+    } else {
+    }
+  };
+
+  const useStyles = makeStyles(theme => ({
+    root: {
+      width: 300 + theme.spacing(3) * 2
+    },
+    margin: {
+      height: theme.spacing(3)
+    }
+  }));
+
+  const MoodSlider = withStyles({
```

Comment:
While not necessary, breaking this component into a separate file will make this file less jumbled